data/projectsModel.js

```diff
@@ -0,0 +1,29 @@
+const db = require("../data/dbConfig.js");
+
+const knex = require("knex");
+
+const config = require("../knexfile.js");
+
```

Comment:
don't think you need the knex and the config here since it's in your dbConfig.js :)